cmds/webboot/webboot_test.go

```diff
@@ -140,8 +141,38 @@ func TestDirOption(t *testing.T) {
 			if iso.label != wanted.label || iso.path != wanted.path {
 				t.Fatalf("Get wrong chosen iso. get %+v, want %+v", iso, wanted)
 			}
+			break
 		} else {
 			t.Fatalf("Unknown type. got entry %+v of type %T, wanted DirOption or ISO", entry, entry)
 		}
 	}
 }
+
+// This test may not work for every one.
+// because the test need a usb stick that contains a /Image folder
+// which structure is
+// +-- Image
+// |  +-- tinycore1
+// |  |  +-- TinyCorePure64.iso
+// |  |  +-- TinyCorePure64 (copy).iso
+// |  +-- tinycore2
+// |     +-- TinyCorePure64-10.1.iso
+// |     +-- TinyCorePure64-10.1 (copy).iso
+// ...
+func TestGetCachedDirectory(t *testing.T) {
```

Comment:
If this test doesn't always pass I'm not sure that we want to include it with the rest of the unit tests...I think with our CI we'd want to run all the unit tests to confirm that they all pass, and if this one always fails that might throw it off? I'm not totally sure how CIs work so maybe this isn't a problem @rminnich

Reply:
I deleted them for now